dist/ramda.js

```diff
@@ -6200,10 +6198,6 @@
     };
 
     var R = {
-        F: F,
-        I: I,
-        T: T,
-        __: __,
```

Comment:
...> because it _appears_ you forgot to update **ramda.js**

Any idea of how that happens when the update is right there in the commit?